lib/resource-providers/rds.ts

```diff
@@ -0,0 +1,55 @@
+import { CfnOutput } from "aws-cdk-lib";
+import * as rds from "aws-cdk-lib/aws-rds";
+import { GlobalResources, ResourceContext, ResourceProvider } from "../spi";
+import {IClusterEngine, IDatabaseCluster, InstanceProps} from "aws-cdk-lib/aws-rds";
+import * as ec2 from "aws-cdk-lib/aws-ec2";
+import {IVpc} from "aws-cdk-lib/aws-ec2";
+
+
+export interface RdsInstanceProps {
+  readonly name?: string;
+  readonly rdsProps?: Omit<rds.DatabaseInstanceProps, "storageEncryptionKey">;
+  readonly rdsEncryptionKeyResourceName?: string;
+}
+
+export interface AuroraClusterProps {
```

Comment:
Jsdoc on each exported member. 
Applies elsewhere.

This feeds our [API documentation](https://aws-quickstart.github.io/cdk-eks-blueprints/api/): 

```
/**
 *  
 */
```

Reply:
Will do it in the next few commits as we further solidify the props